### PR TITLE
Feat - added dialogue component

### PR DESCRIPTION
**Pull request by Mohasin-Haque**
https://deploy-preview-7--fast-ui.netlify.app/  Please check the preview here.

image

### CODE REVIEW

File: styles/dialogue.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+.dialogue-heading {
+  margin-bottom: 1rem;
+}
+
+.dialogue-buttons {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  gap: 1.5rem;
+  margin: 10px 0 0 7rem;
```
Comment (bishnoimukesh) [inline styles/dialogue.css]: either use px or rem

**Review by bishnoimukesh — APPROVED** [review]
one change suggested rest all good to merge

File: styles/dialogue.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+.dialogue-button-yes:hover {
+  font-weight: bold;
+}
+
+.dialogue-button-no {
+  color: green;
+  background-color: white;
+}
+
```
Comment (sanjay-xdr) [inline styles/dialogue.css]: use hex or rgb value for colors

**Review by sanjay-xdr — COMMENTED** [review]
everything looks good

File: styles/dialogue.css (hunk 2 above)
Comment (Mohasin-Haque) [reply styles/dialogue.css]: Done!!

**Pull request merged**